js/alert.js

```diff
@@ -0,0 +1,105 @@
+let renderedAlert;
+let renderedAlertTimeout;
+
+/**
+ * @typedef ButtonParams
+ * @property {string} text
+ * @property {function(Event): void} onClick
+ */
+
+const handleWindowEscKeyDown = (evt) => {
+  if (!evt.code === 'Escape') {
+    return;
+  }
+
+  if (evt.code === 'Escape') {
+    renderedAlert.remove();
+    if (renderedAlertTimeout) {
+      clearTimeout(renderedAlertTimeout);
+
+      renderedAlertTimeout = undefined;
+    }
+  }
```

Comment:
Лишняя проверка на `Escape`, выше уже проверили
```suggestion
    renderedAlert.remove();
    if (renderedAlertTimeout) {
      clearTimeout(renderedAlertTimeout);

      renderedAlertTimeout = undefined;
    }
```